ref/Microsoft.Build/netstandard/Microsoft.Build.cs

```diff
@@ -1377,6 +1377,12 @@ public sealed partial class ProjectTaskOutputPropertyInstance : Microsoft.Build.
         public string TaskParameter { get { throw null; } }
         public override Microsoft.Build.Construction.ElementLocation TaskParameterLocation { get { throw null; } }
     }
+    public sealed partial class RarNode
+    {
+        public RarNode() { }
+        public Microsoft.Build.Execution.NodeEngineShutdownReason Run(bool nodeReuse, bool lowPriority, out System.Exception shutdownException) { shutdownException = default(System.Exception); throw null; }
```

Comment:
minor: please reference System, so that the Exception class does not need full path.